09LessonNine/in-class-project/css/style.css

```diff
@@ -0,0 +1,147 @@
+/* beginning --- index.html style codes*/
+main {
+   text-align: center;
+   width: 100vw;
+   height: 100vh;
+   background-image: url(../img/1.jpg);
+   background-size: cover;
+   font-family: "Lucinda Console";
+}
+
+.capacity {
+   background-color: rgba(0, 0, 0, 0.2);
+   width: 100vw;
+   height: 70vh;
+   padding-top: 30vh;
+   color: white;
+}
+
+.capacity > h4 {
+   font-size: 19pt;
+}
+
+.capacity > h1 {
+   font-size: 50pt;
+}
+
+.capacity > p {
+   font-size: 28pt;
+}
+
+.capacity > a {
+   font-size: 16pt;
+   padding: 10px 25px;
+   background-color: #91B005;
+   color: white;
+   line-height: 3;
+   text-decoration: none;
+}
+/* end --- index.html style codes*/
+
+/*beginning --- service.html sytle codes*/
+
+#container {
+   position: relative;
+   width: 100vw;
+   height: 100vh;
+   font-family: Arial;
+}
+
+.left {
+   position: absolute;
+   width: 37%;
+   height: 700px;
+}
+
+.left h1 {
+   font-size: 32pt;
+   color: white;
+   padding-top: 120px;
+   padding-left: 50px;
+}
+
+.left h4 {
+   font-size: 16pt;
+   color: #91B005;
+   padding-left: 50px;
+   margin-top: -15px;
+}
+
+.right {
+   position: absolute;
+   width: 62.7%;
+   right: 0;
+   height: 700px;
+}
+
+.landscaping {
+   width: 940.5px;
+   height: 665px;
+   margin-top: 20px;
+   background-image: url(../img/featured-image-5.jpg);
+   background-color: rgba(0, 0, 0, 0.5);
+   background-repeat: no-repeat;
+   background-size: contain;
+}
+
+.landscaping h1 {
+   font-size: 40pt;
+   color: white;
+   padding-left: 30px;
+   padding-top: 150px;
+}
+
+.landscaping h4 {
+   font-size: 20pt;
+   color: #91B005;
+   padding-left: 30px;
+}
+
+.landscaping p {
+   font-size: 11pt;
+   height: 700px;
+   color: white;
+   line-height: 200%;
+   padding-left: 30px;
+   padding-right: 300px;
+}
+
+p span a {
+   text-decoration: none;
+   color: white;
+   font-size: 11pt;
+}
+
+span.dark {
+   background-color: black;
+   margin-right: 20px;
+   padding: 13px 20px;
+}
+
+span.whiteline {
+   border-style: solid;
+   padding: 10px 20px;
+}
+
+.growing {
+   width: 455px;
+   height: 320px;
+   margin-left: 239px;
```

Comment:
careful when mixing pixels and percents when addressing the width of content-blocks. also, position: absolute; should not be applied in this project. when i view your project, I see a lot of overlapping content. the heights are correct, but the layout is off.